chainer/links/connection/embed_id.py

```diff
@@ -22,9 +26,12 @@ class EmbedID(link.Link):
         W (~chainer.Variable): Embedding parameter matrix.
 
     """
-    def __init__(self, in_size, out_size):
+
+    def __init__(self, in_size, out_size, initialW=None):
         super(EmbedID, self).__init__(W=(in_size, out_size))
-        self.W.data[...] = numpy.random.randn(in_size, out_size)
+        initializations.init_weight(
+            self.W.data, initialW,
+            none_default=lambda s: initializations.normal(s, scale=1.0))
```

Comment:
I remember that we had discussed whether we should make the initializer a class that takes parameters (`scale` in this case) in `__init__` and invoke initialization with `__call__`. If we use this design, we can use `initializers.Normal(scale)` and avoid using lambda expressions. So could you consider the possibility of this design?